store/build.xml

```diff
@@ -393,6 +393,14 @@
        <arg line="-o ${dist.dir}"/>
     </java>
   </target>
+  <target name="create-version-ldap" depends="build-init" description="Creates ldap schema version: zimbra-attrs-schema.txt">
+    <exec executable="git" failonerror="false" output="conf/attrs/zimbra-attrs-schema">
```

Comment:
As it isn't stored under git I would prefer if this was created under a build/ directory.

Reply:
I'll work on getting this to happen.  I meant to at least add it to the .gitignore file but this is a better idea!